packages/jest-config/src/normalize.ts

```diff
@@ -1139,6 +1139,13 @@ export default async function normalize(
     newOptions.shard = parseShardPair(argv.shard);
   }
 
+  if (argv.randomize) {
+    // at time of writing we use Prando which starts repeating after 2^29 - 1
+    // 0 is not a valid input seed
+    newOptions.seed =
+      argv.seed ?? Math.floor((1 - Math.random()) * (Math.pow(2, 29) - 1));
```

Comment:
we should validate `seed` from user conforms to our contract (integer between `0` and `2**29  - 1` I think?)